index.html

```diff
@@ -457,87 +533,8 @@
-                    </tr>
-                    <tr>
-                        <td><code>pointerup</code></td>
-                        <td>Sync</td>
-                        <td>Yes</td>
-                        <td>Yes</td>
-                        <td>Varies:  when the pointer is primary, all default actions of <code>mouseup</code></td>
-                    </tr>
-                    <tr>
-                        <td><code>pointercancel</code></td>
-                        <td>Sync</td>
-                        <td>Yes</td>
-                        <td>No</td>
-                        <td>None</td>
-                    </tr>
-                    <tr>
-                        <td><code>pointerout</code></td>
-                        <td>Sync</td>
-                        <td>Yes</td>
-                        <td>Yes</td>
-                        <td>None</td>
-                    </tr>
-                    <tr>
-                        <td><code>pointerleave</code></td>
-                        <td>Sync</td>
-                        <td>No</td>
-                        <td>No</td>
-                        <td>None</td>
-                    </tr>
-                    <tr>
-                        <td><code>gotpointercapture</code></td>
-                        <td>Sync/Async</td>
-                        <td>Yes</td>
-                        <td>No</td>
-                        <td>None</td>
-                    </tr>
-                    <tr>
-                        <td><code>lostpointercapture</code></td>
-                        <td>Sync/Async</td>
-                        <td>Yes</td>
-                        <td>No</td>
-                        <td>None</td>
-                    </tr>
-                </tbody>
-            </table>
-            <p>For all pointer events in the table above, <code>composed</code> ([[!WHATWG-DOM]]) attribute SHOULD be <code>true</code> and <a href="https://www.w3.org/TR/uievents/#widl-UIEvent-detail"><code>detail</code></a> [[!DOM-LEVEL-3-EVENTS]] attribute SHOULD be 0.</p>
-            <p>For <code>gotpointercapture</code> and <code>lostpointercapture</code> all the attributes except the ones defined in the table above should be the same as the Pointer Event that caused the user agent to run <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> and fire the <code>gotpointercapture</code> and <code>lostpointercapture</code> events.</p>
+            <p>Below are the event types defined in this specification.</p>
+            <p>All of these events except <code>gotpointercapture</code> and <code>lostpointercapture</code> are fired synchronously; <code>gotpointercapture</code> and <code>lostpointercapture</code> could be fired synchronously or asynchronously. See <a href="https://www.w3.org/TR/uievents/#sync-async">Synchronous and asynchronous events</a> [[!DOM-LEVEL-3-EVENTS]].</p>
```

Comment:
@RByers, @teddink, since we are calling out this sync/async in a separate paragraph I wonder if this is still correct. Do we ever send gotpointercapture or lostpointercapture async in this version of spec? if so can anyone give me an example?